include/CommandHandler/src/CommandHandler.cpp

```diff
@@ -302,20 +307,21 @@ void CommandHandler::HandleVrfy(SocketWrapper& socket_wrapper, const std::string
             Logger::LogWarning("User does not exist: " + user_name);
             socket_wrapper.SendResponseAsync("550 User does not exist\r\n").get();
         }
-    } catch (const MailException& e) {
+    }
+    catch (const MailException& e)
+    {
         Logger::LogError("MailException caught in HandleVrfy: " + std::string(e.what()));
         ErrorHandler::HandleError("Handle VRFY", e, socket_wrapper, "550 Internal Server Error\r\n");
     }
     Logger::LogDebug("Exiting CommandHandler::HandleVrfy");
 }
 
-
 void CommandHandler::HandleExpn(SocketWrapper& socket_wrapper, const std::string& line)
 {
```

Comment:
EXPN command may be redundant because we don't create mailing list on server (if am not mistaken) and they are not stored on database.